apps/cowswap-frontend/src/api/cowProtocol/api.ts

```diff
@@ -1,187 +1,18 @@
-import { ZERO_ADDRESS } from '@cowprotocol/common-const'
-import {
-  isBarn,
-  isDev,
-  isLocal,
-  isPr,
-  isSellOrder,
-  toErc20Address,
-  toNativeBuyAddress,
-} from '@cowprotocol/common-utils'
 import {
   Address,
   CompetitionOrderStatus,
   CowEnv,
   EnrichedOrder,
   NativePriceResponse,
-  OrderQuoteRequest,
-  OrderQuoteResponse,
-  OrderQuoteSideKindBuy,
-  OrderQuoteSideKindSell,
   PartialApiContext,
-  SigningScheme,
   SupportedChainId as ChainId,
   TotalSurplus,
 } from '@cowprotocol/cow-sdk'
 
 import { orderBookApi } from 'cowSdk'
 
-import { getAppData } from 'modules/appData'
-
-import { FeeQuoteParams } from 'common/types'
-import { getQuoteValidFor } from 'utils/orderUtils/getQuoteValidFor'
-
-import { ApiErrorCodes } from './errors/OperatorError'
-import { QuoteApiError, mapOperatorErrorToQuoteError, QuoteApiErrorDetails } from './errors/QuoteError'
-import { getIsOrderBookTypedError } from './getIsOrderBookTypedError'
-
 export { getIsOrderBookTypedError } from './getIsOrderBookTypedError'
 
-function getProfileUrl(): Partial<Record<ChainId, string>> {
-  if (isLocal || isDev || isPr || isBarn) {
-    return {
-      [ChainId.MAINNET]:
-        process.env.REACT_APP_PROFILE_API_URL_STAGING_MAINNET || 'https://barn.api.cow.fi/affiliate/api',
-    }
-  }
-
-  // Production, staging, ens, ...
-  return {
-    [ChainId.MAINNET]: process.env.REACT_APP_PROFILE_API_URL_STAGING_MAINNET || 'https://api.cow.fi/affiliate/api',
-  }
-}
-
-const PROFILE_API_BASE_URL = getProfileUrl()
-
-const DEFAULT_HEADERS = {
-  'Content-Type': 'application/json',
-}
-const API_NAME = 'CoW Protocol'
-
-function _getProfileApiBaseUrl(chainId: ChainId): string {
-  const baseUrl = PROFILE_API_BASE_URL[chainId]
-
-  if (!baseUrl) {
-    throw new Error(`Unsupported Network. The ${API_NAME} API is not deployed in the Network ` + chainId)
-  } else {
-    return baseUrl + '/v1'
-  }
-}
-
-function _fetchProfile(
-  chainId: ChainId,
-  url: string,
-  method: 'GET' | 'POST' | 'DELETE',
-  // TODO: Replace any with proper type definitions
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  data?: any,
-): Promise<Response> {
-  const baseUrl = _getProfileApiBaseUrl(chainId)
-
-  return fetch(baseUrl + url, {
-    headers: DEFAULT_HEADERS,
-    method,
-    body: data !== undefined ? JSON.stringify(data) : data,
-  })
-}
-
-function _getProfile(chainId: ChainId, url: string): Promise<Response> {
-  return _fetchProfile(chainId, url, 'GET')
-}
-
-// ETH-FLOW orders require different quote params
-// check the isEthFlow flag and set in quote req obj
-const ETH_FLOW_AUX_QUOTE_PARAMS = {
-  signingScheme: SigningScheme.EIP1271,
-  onchainOrder: true,
-  // Ethflow orders are subsidized in the backend.
-  // This means we can assume the verification gas costs are zero for the quote/fee estimation
-  verificationGasLimit: 0,
-}
```

Comment:
This is all captured on the SDK, right?

Reply:
Yes sir
https://github.com/cowprotocol/cow-sdk/blob/860ff870f3e008552df57ad527cb48371b1065a8/src/trading/getQuote.ts#L36